src/client/components/terminal/index.ts

```diff
@@ -305,6 +305,7 @@ export class TerminalView extends LitElement {
   protected windowSize: IWindowSize
 
   protected rows: number = 10
+  protected errorCode?: any
```

Comment:
We know the type of this one right?
```suggestion
  protected errorCode?: number
```